_docs/en/connect-a-data-source/plugins/110-s3-storage-plugin.md

```diff
@@ -14,7 +14,7 @@ To connect Drill to S3:
 
 For additional information, refer to the [HDFS S3 documentation](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html).   
 
-**Note:** Drill does not use HDFS 3.x, therefore Drill does not support AWS temporary credentials, as described in the s3a documentation.
+**Note:** Drill started using HDFS 3.0, but support of AWS temporary credentials (as described in the s3a documentation) wasn't verified yet.
```

Comment:
This is confusing.  Does Drill support temporary creds?  If not should we?  I'd think that's something users would want.

Reply:
This about AWS temporary creds, not Drill temporary creds.

I just found outdated info, the doc was written before Drill supports hadoop3.0.
But now Drill leverages `3.2.2` hadoop version. And it supports `TemporaryAWSCredentialsProvider` 
```
<property>
  <name>fs.s3a.session.token</name>
  <description>
    Session token, when using org.apache.hadoop.fs.s3a.TemporaryAWSCredentialsProvider
    as one of the providers.
  </description>
</property>
```
https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html
Therefore  it can be used in Drill.